pkg/cmd/server/api/types.go

```diff
@@ -1407,3 +1408,16 @@ type DefaultAdmissionConfig struct {
 	// Disable turns off an admission plugin that is enabled by default.
 	Disable bool
 }
+
+// TemplateServiceBrokerConfig holds information related to the template
+// service broker
+type TemplateServiceBrokerConfig struct {
+	// Enabled switches on the types, API and controllers associated with the
+	// template service broker
+	Enabled bool
+
+	// TemplateNamespaces indicates the namespace(s) in which the template service
+	// broker looks for templates to serve.  If empty, the default is the
```

Comment:
> then why is the openshift namespace accessible to all users when i simply bring up an openshift master w/ no configuration file?

We create a default config that has it set.  This action is different (and what I suggested as an option), than using an actual deserialization default value.  Specifying it in a default config allows someone to clear it (doesn't steal API power) as opposed to a deserialization default value which does not allow someone to clear it (does steal API power).

Specifying it in a default configuration doesn't require much thought since it doesn't really affect your API.  Specifying it as a deserialization default value needs to be carefully considered since it locks your API into a position of having some values be unexpressable.